**Background**

In quality control processes, especially when dealing with high-value items, destructive sampling is a necessary but costly method to ensure product quality. The test to determine whether an item meets the quality standards destroys the item, leading to the requirement of small sample sizes due to cost constraints.


**Scenario**

A manufacturer of print-heads for personal computers is interested in estimating the mean durability of their print-heads in terms of the number of characters printed before failure. To assess this, the manufacturer conducts a study on a small sample of print-heads due to the destructive nature of the testing process.

**Data**

A total of 15 print-heads were randomly selected and tested until failure. The durability of each print-head (in millions of characters) was recorded as follows:
1.13, 1.55, 1.43, 0.92, 1.25, 1.36, 1.32, 0.85, 1.07, 1.48, 1.20, 1.33, 1.18, 1.22, 1.29

In [1]:
import numpy as np
import scipy.stats as st

In [4]:
data=np.array([1.13, 1.55, 1.43, 0.92, 1.25, 1.36, 1.32, 0.85, 1.07, 1.48, 1.20, 1.33, 1.18, 1.22, 1.29])

In [6]:
mean=np.mean(data)
std=np.std(data)

In [7]:
print("Mean:", mean)
print("Standard Deviation:", std)

Mean: 1.2386666666666666
Standard Deviation: 0.18661427836285438


**Assignment Tasks**

a. Build 99% Confidence Interval Using Sample Standard Deviation
Assuming the sample is representative of the population, construct a 99% confidence interval for the mean number of characters printed before the print-head fails using the sample standard deviation. Explain the steps you take and the rationale behind using the t-distribution for this task.

Mean: 1.2386666666666666

Standard Deviation: 0.18661427836285438

CL : 0.99

n=15

df (degree of freedom) = n-1 = 15-1 =14

We will use t-method

In [12]:
st.t.interval(0.95,15,loc=mean,scale=std/np.sqrt(15))

(1.1359657554425353, 1.3413675778907979)

b. Build 99% Confidence Interval Using Known Population Standard Deviation

If it were known that the population standard deviation is 0.2 million characters, construct a 99% confidence interval for the mean number of characters printed before failure.

Mean: 1.2386666666666666

Standard Deviation: 0.2 (given)

CL : 0.99

n=15

We will use z-method since standard deviation is known

In [14]:
st.norm.interval(0.99,mean,0.2/np.sqrt(15))

(1.1056514133957607, 1.3716819199375725)